p2p/p2p.go

```diff
@@ -67,17 +67,19 @@ func NewTCPNode(cfg Config, key *ecdsa.PrivateKey, connGater ConnGater,
 		// Limit connections to DV peers.
 		libp2p.ConnectionGater(connGater),
 
+		libp2p.AddrsFactory(factory),
```

Comment:
So turns out libp2p also does a form of peer discovery. We should turns this off since we use discv5 for peer discovery. See two factories above.